src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -455,17 +477,19 @@
 					x:Load="{x:Bind ViewModel.DriveCapacityVisibility, Mode=OneWay}"
 					FontWeight="SemiBold"
 					Text="{helpers:ResourceString Name=PropertiesDriveCapacity/Text}" />
+
 				<TextBlock
 					x:Name="DriveCapacityValue"
 					Grid.Row="2"
 					Grid.Column="2"
 					x:Load="{x:Bind ViewModel.DriveCapacityVisibility, Mode=OneWay}"
 					Style="{StaticResource PropertyValueTextBlock}"
 					Text="{x:Bind ViewModel.DriveCapacity, Mode=OneWay}" />
+
 				<Button
 					Grid.Row="2"
 					Grid.Column="3"
-					Padding="8,6"
+					Padding="6,6"
```

Comment:
Can you share a screenshot of the before and after?

Reply:
As above